ci-operator/step-registry/hypershift/agent/create/config-dns/hypershift-agent-create-config-dns-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ config-dns command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
+echo "$CLUSTER_NAME" > /tmp/hostedcluster_name
+scp "${SSHOPTS[@]}" "/tmp/hostedcluster_name" "root@${IP}:/home/hostedcluster_name"
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF |& sed -e 's/.*auths.*/*** PULL_SECRET ***/g'
+MASTER_NUM=\$(oc get node -lnode-role.kubernetes.io/master="" --no-headers | wc -l)
+HOSTEDCLUSTER_NAME=\$(cat /home/hostedcluster_name)
+echo "address=/api.\$HOSTEDCLUSTER_NAME.ostest.test.metalkube.org/192.168.111.\$((21+\$MASTER_NUM))" >> /etc/NetworkManager/dnsmasq.d/openshift-ostest.conf
```

Comment:
I think it's better if you just take the IP of one of the cluster nodes:
```
oc get nodes master-0 -o jsonpath='{.status.addresses[0].address}'
```

Reply:
Sure, I think we can use this method, but can it use the master IP? I remember I made this attempt, worker ip is ready, but master ip is not to be ready

Reply:
Worker should be better